src/Notes.cc

```diff
@@ -124,6 +125,30 @@ Note::toString(const char *sep) const
     return result;
 }
 
+static const char *NotesBlackListStr[] = {
+    "user",
+    "group",
+    "password",
+    "status",
+    "message",
+    "log",
+    "tag",
+    "ttl",
+    "ha1",
+    "rewrite-url",
+    "url"
+};
+const Notes::KeysList Notes::BlackList(std::begin(NotesBlackListStr), std::end(NotesBlackListStr));
+
+Notes::Notes(const char *aDescr, const Notes::KeysList *metasBlacklist, bool allowFormatted): descr(aDescr), blacklisted(Notes::BlackList), formattedValues(allowFormatted)
```

Comment:
This being a long line, please use multi-line form for the initialization list entries:

```suggestion
Notes::Notes(const char *aDescr, const Notes::KeysList *metasBlacklist, bool allowFormatted) :
        descr(aDescr),
        blacklisted(Notes::BlackList),
        formattedValues(allowFormatted)
```

Reply:
commit de7ed8eb